### PR TITLE
Update copy on OpenStack page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update copy on /openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the page has been updated according to [the copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#) and resolve the comments in the doc

## Issue / Card

Fixes #3066

### CODE REVIEW

**Review by pmahnke — APPROVED** [review]
+1

File: templates/shared/pricing/_openstack-packages.html [hunk 1]
```diff
@@ -18,27 +17,29 @@ <h2 class="p-heading--four">Private Cloud Build</h2>
...
+    <p>Design and delivery on certified hardware.</p>
+    <p><a href="/openstack/consulting">Get OpenStack&nbsp;&rsaquo;</a></p>
   </div>
   <div class="col-{{ 12 // columns }} p-card">
     <h2 class="p-heading--four">Private Cloud Build Plus</h2>
-    <p><span class="p-heading--two">$150,000</span><br />Custom architecture for your specific industry or workload requirements.</p>
+    <p><span class="p-heading--two">$150,000</span> fixed price</p>
     <hr class="u-sv1" />
-    <p>Co-designed workshops to determine the optimal architecture for your workloads and integration requirements. Our engineers then deploy it for you.</p>
+    <p>Design and Deployment of a secure, carrier grade, feature rich OpenStack custom architecture</p>
```
Comment (pmahnke) [inline templates/shared/pricing/_openstack-packages.html:27]: missing full-stop at the end
```suggestion
    <p>Design and Deployment of a secure, carrier grade, feature rich OpenStack custom architecture.</p>
```

**Review by pmahnke — APPROVED** [review]
one comment, but it looks merged

File: templates/shared/pricing/_openstack-packages.html (hunk 1 above)
Comment (steverydz) [reply templates/shared/pricing/_openstack-packages.html:27]: Fixed here https://github.com/canonical-web-and-design/ubuntu.com/pull/8129

**Pull request merged**